### ISSUE TITLE
(Bug) proveBalanceOfAtBlockchain not updated for chain id ?????

### ISSUE THREAD

**Issue by johnsmith-gooddollar** [GoodDollar/GoodProtocolUI#361]
![image.png](https://images.zenhubusercontent.com/6135d0416eed5f7809f4ee6b/a74f4420-880c-431c-a18b-3788b3ea854b)

```
Error: update failed:: Invalid number of parameters for “proveBalanceOfAtBlockchain”. Got 5 expected 4!
```

looks like old protocol/contracts are used (which do not support chainId)
but package.json shows the same version 2.0.3 like at the protocolUI

**Comment by sirpy** [comment]
@johnsmith-gooddollar this is not related to /GoodDAPP/issues/3897

**Comment by sirpy** [comment]
fixed in GoodDAPP 0c144845e385460d483a687fed5035dcc86186da

**Comment by vldkhh** [comment]
checked via #3897
